src/lib/utils/update.ts

```diff
@@ -1,13 +1,25 @@
+import DynamoDBSet from 'aws-sdk/lib/dynamodb/set';
+import { Map, UpdateQuery } from '../types';
 import * as nameUtil from './name';
-import { UpdateQuery, Map } from '../types';
 
 interface ParseResult {
 	UpdateExpression: string;
 	ExpressionAttributeNames: Map<string>;
 	ExpressionAttributeValues: Map<any>;
 }
 
-export const operators = ['$set', '$unset', '$inc', '$push', '$unshift'];
+export const operators = ['$set', '$unset', '$inc', '$push', '$unshift', '$addToSet'];
+
+const fromArrayEach = (value) => {
+	if (value.$each) {
+		if (!Array.isArray(value.$each)) {
+			throw new Error('The value for $each should be an array.');
+		}
+		return value.$each;
+
+	}
+	return [value];
```

Comment:
Small nitpick comment :).

```suggestion
	if (value.$each) {
		if (!Array.isArray(value.$each)) {
			throw new Error('The value for $each should be an array.');
		}

		return value.$each;
	}

	return [value];
```